README.md

```diff
@@ -1,24 +1,13 @@
-# Personal Portfolio Site
+# Group Fansite
 
 ## Introduction
 
-It is common for developers to create a website to show off their web-dev skills and document their work and interests. These sites often feature things like:
+When first starting out, it is common for developers to create a small website to show off their web-dev skills and document their interests. 
 
-- A professional introduction of themselves, careers, and interests
-- A portfolio of projects that they've created and/or collaborated on
-- A blog to share their experiences, thoughts, and/or subject matter expertise
-- Other materials to share any non-coding interests and hobbies
-- A site designed to express themselves
+We will use HTML & CSS to create a static site whose content is a fansite about something your group is passionate about. Your group fansite should contain a home page with some introductory information about your chosen topic, fun facts about your topic, and pictures showcasing your topic. 
```

Comment:
This gets clarified later, but I had the thought that it might be helpful to be clear here that we're describing 3 separate pages and not a home page with all of these features. Maybe something like:

> Your group fansite should contain a home page with some introductory information about your chosen topic, a fun facts page about your topic, and a gallery showcasing pictures about your topic.